Containerfile

```diff
@@ -1,5 +1,5 @@
 # === Build fixtures (Fedora) =================================================
-FROM fedora:30 AS fedora-build
+FROM fedora:latest AS fedora-build
```

Comment:
we had some issues on pulp_installer, we had to specify the registry:
https://github.com/pulp/pulp_installer/blob/master/molecule/scenario_resources/Dockerfile.j2#L9